software-modules/programming/neovim/neovim.desktop

```diff
@@ -0,0 +1,81 @@
+[Desktop Entry]
+Name=Neovim
+GenericName=Text Editor
+GenericName[de]=Texteditor
+Comment=Edit text files
+Comment[af]=Redigeer tekslêers
```

Comment:
Should we keep the translations? the UI is supposed to be english as standard of IOI and ICPC. And I don't see close to start supporting several UI langs.

Reply:
Nah, let me remove the translations.